code/modules/jobs/job_types/security.dm

```diff
@@ -191,6 +192,7 @@ Security Officer
 	supervisors = "the head of security, and the head of your assigned department (if applicable)"
 	selection_color = "#ffeeee"
 	minimal_player_age = 7
+	alt_titles = list("Security Guard", "Marine")

```

Comment:
I dunno about these. Maybe rename security officer to marine or just remove the security guard one.